src/biogeophys/OzoneMod.F90

```diff
@@ -494,21 +640,21 @@ subroutine CalcOzoneUptakeOnePoint( &
        endif
 
        if (pftcon%evergreen(pft_type) == 1) then
-          leafturn = 1._r8/(pftcon%leaf_long(pft_type)*365._r8*24._r8)
+          leafturn = 1._r8/(pftcon%leaf_long(pft_type)*avg_dayspyr*secspday)
        else
           leafturn = 0._r8
        endif
 
        ! o3 uptake decay based on leaf lifetime for evergreen plants
-       decay = o3uptake * leafturn * dtimeh
+       decay = o3uptake * leafturn * dtime
```

Comment:
Is this OK to do? Because dtimeh is in hours and dtime is in seconds, won't this change answers for the older versions?

Reply:
According to Sam's suggestion, leafturn is changed from per hour to per second. Correspondingly, we should use dtime (in seconds) instead of dtimeh (in hours).